File: Kotlin/FizzBuzz/fizzbuzz.kt
fun fizzbuzz(n: Int) {
    for (i in 1..n) {
        if (i % 15 == 0) {
            println("FizzBuzz")
        } else if (i % 3 == 0) {
            println("Fizz")
        } else if (i % 5 == 0) {
            println("Buzz")
        } else {
            println(i)
        }
    }
}

fun main(args: Array<String>) {
    fizzbuzz(15)
}

File: Lua/FizzBuzz/fizzbuzz.lua
function fizzbuzz(n)
   local i = 1;
   while(i <= n)
   do
      if(i % 15 == 0) then
         print("FizzBuzz")
      elseif(i % 3 == 0) then
         print("Fizz")
      elseif(i % 5 == 0) then
         print("Buzz")
      else
         print(i)
      end
      i = i + 1;
   end
end

fizzbuzz(15)
